core/client/src/client.rs

```diff
@@ -369,13 +369,17 @@ impl<B, E, Block> Client<B, E, Block> where
 		);
 		self.state_at(&parent).and_then(|state| {
 			let mut overlay = Default::default();
-			let execution_manager = || ExecutionManager::Both(|wasm_result, native_result| {
-				warn!("Consensus error between wasm and native runtime execution at block {:?}", at);
-				warn!("   Function {:?}", function);
-				warn!("   Native result {:?}", native_result);
-				warn!("   Wasm result {:?}", wasm_result);
-				wasm_result
-			});
+			let execution_manager = || match self.execution_strategy {
```

Comment:
what's the reasoning behind this?

Reply:
There's a config option, why not use it here and give user control?

Reply:
i think the config option is for when syncing the chain or otherwise not authoring. this is block production stuff so we want to remain as safe as possible. if my assertion is correct then we'll at least want to place it under another config option.

Reply:
Added a second config option.